docs/guide/cli-reference.md

```
@@ -555,6 +555,19 @@ List workspace members.
 cmod workspace list
 ```
 
+### `cmod registry validate`
+
+Validate a registry `index.json` against governance policy (the same rules
+`cmod publish` submissions face). Used by the cmod-registry validation CI.
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)